site-src/concepts/api-overview.md

```diff
@@ -157,7 +159,7 @@ to allow multiple Routes to share ports on the Listener.
 |------|---------|---------------------|-----------|-------|
 |HTTPRoute| Layer 7 | Anything in the HTTP Protocol | Terminated only | HTTP and HTTPS Routing|
 |TLSRoute| Somewhere between layer 4 and 7| SNI or other TLS properties| Passthrough or Terminated | Routing of TLS protocols including HTTPS where inspection of the HTTP stream is not required.|
-|TCPRoute| Layer 4| destination port | Passthrough or Terminated | Allows for forwarding of a TCP stream from the Listener to the Backends |
+|TCPRoute| Layer 4| destination port | Terminated | Allows for forwarding of a TCP stream from the Listener to the Backends |
```

Comment:
I think this is correct and in alignment with the current docs for https://gateway-api.sigs.k8s.io/reference/spec/#tlsmodetype and clarifications in https://gateway-api.sigs.k8s.io/geps/gep-2907/#6-configure-tls-mode.

I believe TCPRoute can still be attached to a listener _omitting_ any TLS configuration, which can be an alternative means of passthrough for TLS connections where routing based on hostname is not needed.

Reply:
there is no need to support `TCPRoute` with `mode: Passthrough`. When a `TCPRoute` is used, the GW Listener should use `protocol: TCP`.
- A TCP Listener passes TLS traffic through by default, `mode: Passthrough` is unnecessary.
- `mode: Passthrough` is only needed when you need to match TLS-specific metadata (e.g. using `TLSRoute`) without offloading the TLS.